### DISCUSSION TITLE
try not to say big chungus challenge

### DISCUSSION THREAD

**Discussion by davwat** [CluckCluckChicken/Magnifier#15] [big chungus]
dont say it guys 😳

**Comment by return-misaPuding** [comment]
big chungu... noooo i cant say this!!!111!!1

**Comment by JackyBorderCollie** [comment]
bi............ oh wait Jacky WHAT ARE YOU DOING YOU'RE NOT SUPPOSED TO SAY THAT!!!!!1!1!!!!!1!

**Comment by FunctionalMetatable** [comment]
big chu...........................................................................
oof

**Comment by ItsEnderium** [comment]
it's funny because it's in the topic title and it's the name of this forum
also we need a tbgs
